.travis.yml

```diff
@@ -8,10 +8,18 @@ compiler:
   - gcc
   - clang
 
+env:
+  - CMAKE_FLAGS=
+  - CMAKE_FLAGS=-DTINYSPLINE_DOUBLE_PRECISION=YES
+  - CMAKE_FLAGS=-DTINYSPLINE_DISABLE_CXX11_FEATURES=YES
+  - CMAKE_FLAGS=-DTINYSPLINE_DOUBLE_PRECISION=YES -DTINYSPLINE_DISABLE_CXX11_FEATURES=YES
```

Comment:
I think we need to quote this exports as well due to the `=` after `-DTINYSPLINE_DOUBLE_PRECISION` and `-DTINYSPLINE_DISABLE_CXX11_FEATURES`. As you can see, the build configuration `-DTINYSPLINE_DOUBLE_PRECISION=YES -DTINYSPLINE_DISABLE_CXX11_FEATURES=YES` only uses `-DTINYSPLINE_DOUBLE_PRECISION`, `-DTINYSPLINE_DISABLE_CXX11_FEATURES=YES` is ignored.